lib/Value.pm

```diff
@@ -353,7 +353,9 @@ sub classMatch {
   my $self = shift;
   return $self->classMatch(@_) if Value->subclassed($self,"classMatch");
   my $class = class($self)//''; my $ref = ref($self);
-  my $isHash = ($ref && $ref ne 'ARRAY' && $ref ne 'CODE');
+  #my $isHash = ($ref && $ref ne 'ARRAY' && $ref ne 'CODE');
+  #warn "hash $isHash $ref";
```

Comment:
Remove these comments (they just make things harder to read, and the old code is in the GitHub history, so no need to keep it here).

Reply:
ok. comments removed.